test/images/iperf/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM fedora
+MAINTAINER jay@apache.org
+RUN dnf install -y git
+RUN git clone https://github.com/esnet/iperf.git
+RUN cd iperf; git checkout 2.0.4-RELEASE 
+RUN dnf install -y gcc
+RUN dnf install -y gcc-c++
+RUN dnf install -y make
+RUN cd iperf && ./configure && make && make install
+RUN rm -rf iperf
```

Comment:
wasnt really doing this for compression, more making it an idiomatic  microservice, but i agree its good to actually remove it b4 it becomes a layer